include/psa/crypto_extra.h

```diff
@@ -663,6 +663,77 @@ mbedtls_ecp_group_id mbedtls_ecc_group_of_psa( psa_ecc_family_t curve,
 
 /**@}*/
 
+/** \defgroup audit Key store audit
+ * @{
+ *
+ * \note    The audit interface is experimental and subject to change
+ *          without notice.
+ */
+
+/** Audit flag indicating that the key material was never and will never
+ * be exposed in plaintext form outside security boundary of its location.
+ *
+ * This flag should be set if all of the following conditions are met:
+ * - The key was generated randomly with psa_generate_key().
+ * - The key had the flag #PSA_KEY_USAGE_EXPORT from the moment of creation.
```

Comment:
Inverted: This should be "the key has not had the flag ..." or "the key was created without #PSA_KEY_USAGE_EXPORT" or similar.

Reply:
Thanks. I fixed this and also mistakes related to copied keys.